File: code/C1Sumlen.java
/**
* cd code
* javac C1Sumlen.java
* java C1Sumlen
*/
public class C1Sumlen {

	public static void main(String[] args) {
		int sl = sumlen("hello", "world");
		System.out.println(sl);
	}

	static int sumlen(String x, String y) {
		return x.length() + y.length();
	}
}

File: code/C1Sumlen.js
/**
* 如何运行我？
*  方式-1： node C1Sumlen.js
*  方式-2： 访问 jstest.html，可以用浏览器，也可以 http-server (node模块)
*/

function sumlen(x, y) {
    return x.length + y.length;
}

var sl = sumlen("hello", 'world');
console.log(sl);
